src/components/AccountDrawer/MiniPortfolio/Activity/OffchainActivityModal.tsx

```diff
@@ -157,19 +167,25 @@ export function OrderContent({ order }: { order: SelectedOrderInfo }) {
       return (
         <ContentContainer>
           <ErrorContent />
-          <ThemedText.SubHeaderLarge>{t`Cancelled`}</ThemedText.SubHeaderLarge>
-          <ThemedText.LabelSmall textAlign="center">{t`This order was cancelled`}</ThemedText.LabelSmall>
+          <ThemedText.SubHeaderLarge>
+            <Trans>Cancelled</Trans>
+          </ThemedText.SubHeaderLarge>
+          <ThemedText.LabelSmall textAlign="center">
+            <Trans>This order was cancelled</Trans>
+          </ThemedText.LabelSmall>
         </ContentContainer>
       )
     case UniswapXOrderStatus.EXPIRED:
       return (
         <ContentContainer>
           <ErrorContent />
-          <ThemedText.SubHeaderLarge>{t`Swap expired`}</ThemedText.SubHeaderLarge>
+          <ThemedText.SubHeaderLarge>
+            <Trans>Swap expired</Trans>
+          </ThemedText.SubHeaderLarge>
           <DescriptionText>
-            {t`Your swap expired before it could be filled. Try again or`}{' '}
+            <Trans>Your swap expired before it could be filled. Try again or</Trans>{' '}
             <LearnMoreLink href="https://support.uniswap.org/hc/en-us/articles/17515426867213">
-              {t` learn more`}.
+              <Trans>learn more.</Trans>
             </LearnMoreLink>
```

Comment:
This must be included as one `Trans`, or it may not translate correctly. I'm not sure what the best path forward is. Possibly making the entire thing a link.

Reply:
oof this one is tricky - i'd be okay with leaving it maybe weirdly translated for now and adding a TODO + linear ticket here to revisit

Reply:
As part of Q3 we want to overhaul our site to be more translation friendly. This could be included as part of that work. cc: @JackShort